components/FaqPage/RenderLine.tsx

```diff
@@ -0,0 +1,32 @@
+// Simple sanitization function that only allows <strong> tags
+const sanitizeHTML = (text: string): string => {
+  return text
+    .replace(/<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi, '')
+    .replace(/javascript:/gi, '')
+    .replace(/on\w+\s*=/gi, '')
+    .replace(/<[^>]*>/g, (match) => {
+      // Only allow <strong> tags
+      if (match.toLowerCase() === '<strong>' || match.toLowerCase() === '</strong>') {
+        return match;
+      }
+      return '';
+    });
+};
```

Comment:
Single Responsibility Principle
- actual: more than one function is defined in this file
- required: precisely one function is defined in each file
Please make a new lib for `/lib/faq/sanitizeHTML`